views/components/overlay-text/macro.njk

```diff
@@ -0,0 +1,7 @@
+{% macro overlayText(text, linkUrl = false, accent = 'pink', isTransluscent = false) %}
+    <span class="overlay-text overlay-text--accent-{{ accent }}{% if isTransluscent %} overlay-text--transluscent{% endif %}">
```

Comment:
The biggest change here was to refactor this so that it doesn't require a heading level (this macro is referenced with heading instead).